### ISSUE TITLE
shutdown bug:

### ISSUE THREAD

**Issue by FrostKnight** [lumina-desktop/lumina#715]
1.5.0 Lumina Desktop seems to logout when I go to poweroff. Same with restart as well.

**Comment by beanpole135** [comment]
Please run "lumina-info" and see what the "OS Build" version is for your package.

My guess is that it is using the "Linux" (Generic Linux) template file for the OS-interaction hooks, and the poweroff/reboot commands listed in that template file do not match what is expected by Hyperbola:
https://github.com/lumina-desktop/lumina/blob/master/src-qt5/core/libLumina/LuminaOS-Linux.cpp#L153

Creating a custom template file just for hyperbola with the correct OS hooks would probably be the best course of action.

**Comment by FrostKnight** [comment]
yeah its Linux.
Hmm... I dont even know how to do that though.  :/

**Comment by beanpole135** [comment]
Marking this issue as "Help Wanted" (by somebody from Hyperbola in this case).
Need somebody using that OS to provide the OS-specific definitions:

OS-Detection file: https://github.com/lumina-desktop/lumina/blob/master/src-qt5/OS-detect.pri
OS-definitions files are called LuminaOS-[OS Name].cpp here: https://github.com/lumina-desktop/lumina/tree/master/src-qt5/core/libLumina

**Comment by FrostKnight** [comment]
Yeah, it is is a bit odd, I assume no one else is running into the same problems?

I wonder if it is an openrc issue if thats the case. :/

**Comment by rjzins** [comment]
I have this same problem when trying to shutdown or restart using the "Leave" menu when using Lumina on Void Linux - it logs out instead of shutting down or restarting. I'm not sure this is isolated to Hyperbola. I have to shutdown or reboot using terminal commands, the leave menu won't do it.

**Comment by rjzins** [comment]
Adding to my previous comment: I don't think this is Hyperbola related. I noticed recently that when I attempt shutdown or restart (Lumina desktop on Void Linux), a polkit type dialog box pops up, asking for a password. The screen then exits and goes to the greeter. It was happening so fast I wasn't seeing the dialog before, my machine was running slow and I saw it recently. Unfortunately it is still happening too fast to screenshot the dialog or to react to it. So this seems to be more generic than just hyperbola.
